cdap-watchdog/src/main/java/co/cask/cdap/logging/context/LoggingContextHelper.java

```diff
@@ -199,29 +206,4 @@ public static Filter createFilter(LoggingContext loggingContext) {
 
     }
   }
-
-  private static Filter createGenericFilter(String namespaceId, String applicationId, String entityId) {
```

Comment:
It is not needed as `GenericLoggingContext` is not used like other logging contexts.